src/getPropsInfoList.ts

```diff
@@ -16,6 +21,11 @@ function getPropsInfoList(component: RuntimeComponentOptions): PropInfo[] {
 
   return Object.keys(props).map(name => {
     const prop = (props as any)[name]
+    let desc = ''
```

Comment:
Would you change this to `description` and use shorthand property names in [L37](https://github.com/pocka/storybook-addon-vue-info/pull/46/files#diff-e9020958da64aa897d3729ec7fd55bcfR37)& [L46](https://github.com/pocka/storybook-addon-vue-info/pull/46/files#diff-e9020958da64aa897d3729ec7fd55bcfR46) ?

Reply:
Sure. I changed it and will be committed